docs/using-kontena/logs.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Logs
+---
+
+# Logs
+
+Kontena streams all logs of the services deployed and stores them in Kontena Master for easy access.
+
+There are various methods to access this log information.
+
+- `kontena grid logs` for grid wide logs (all services and Kontena internal components)
+- `kontena service logs xyz` for individual service logs
+- `kontena stack logs xyz` for stacks service logs
+
+The logs are stored in capped collection which ensures that old events gets automatically removed and do not consume disk space on master database.
```

Comment:
The logs are stored in a capacity-bounded MongoDB collection, which limits the disk space consumed on the master by automatically removing old logs.